capstone-app/src/app/modules/maps/map/map.component.ts

```diff
@@ -27,47 +42,122 @@ export class MapComponent {
     setTimeout(() => {
       this.locationSearch();
     });
+    this.nearSearch = new google.maps.places.PlacesService(this.mapComponent._googleMap);
   }
 
   locationSearch() {
     const input = this.searchBar.nativeElement;
+    this.cityLocation = this.searchBar.nativeElement.value;
```

Comment:
I think you can use NgModel instead of parsing the DOM for the value
Example: https://material.angular.io/components/input/examples